readme.txt

```diff
@@ -31,6 +31,13 @@ Astoundify has released two themes that are fully integrated with the WP Job Man
 
 == Changelog ==
 
+= 1.17.5: Oct 27, 2020 =
+
+* Fix: Minify JS file.
+* Update: Compatibility check with latest WordPress v5.5.1.
+* Update: Compatibility check with Latest WP Job Manager v1.34.3.
+* Update: Compatibility check with the latest PHP v7.4.10.
```

Comment:
Please mention in changelog about https://github.com/Astoundify/wp-job-manager-regions/pull/66.

Reply:
@rosinghal update a few days ago.